llama_stack/distribution/build_container.sh

```diff
@@ -132,6 +134,9 @@ fi
 
 set -x
 $DOCKER_BINARY build $DOCKER_OPTS -t $image_name -f "$TEMP_DIR/Dockerfile" "$REPO_DIR" $mounts
+
+# clean up tmp/configs
+rm -rf $REPO_CONFIGS_DIR
```

Comment:
what creates the directory?

Reply:
We store build.yaml / run.yaml needed to be baked in docker image in $REPO_CONFIGS_DIR, which can be cleaned up after docker is built.

Reply:
right -- I just meant what creates the `tmp/configs` directory? I see this `rm -rf`, but didn't see the corresponding `mkdir`.

Reply:
https://github.com/meta-llama/llama-stack/blob/d74501f75cdea8d59bde2acc695a50cd634a9d94/llama_stack/cli/stack/build.py#L116-L120 - this is where we have mkdir.

Reply:
ah ha, thank you! I searched for the wrong string!